.github/templates/main.yml

```diff
@@ -104,6 +115,11 @@ run: |
 shell: bash
 #@ end
 
+#@ def buildNetFrameworkTest():
+name: Build the tests
+run: #@ "MSBuild.exe ${{ github.workspace }}/Tests/Realm.Tests -p:RestoreConfigFile=${{ github.workspace }}/Tests/Test.Nuget.Config -p:TargetFramework=net461 -p:Configuration=" + configuration + " -p:UseRealmNupkgsWithVersion=${{ needs.build-packages.outputs.package_version }} -restore"
```

Comment:
My suggestion was to extract the msbuild invocation as a separate function that can be used elsewhere - e.g. in `buildPackages`. It can streamline the execution by e.g. accepting a relative path to the project, inserting the configuration, etc.